test/e2e/utils.go

```diff
@@ -628,6 +635,44 @@ func purgeSnapstore(store brtypes.SnapStore) error {
 	return nil
 }
 
+func getPurgeLocalSnapstoreJob(storeContainer string) *batchv1.Job {
+	directory := corev1.HostPathDirectory
+
+	return newTestHelperJob(
+		"purge-local-snapstore",
+		&corev1.PodSpec{
+			Volumes: []corev1.Volume{
+				{
+					Name: "host-dir",
+					VolumeSource: corev1.VolumeSource{
+						HostPath: &corev1.HostPathVolumeSource{
+							Path: "/etc",
+							Type: &directory,
+						},
+					},
+				},
+			},
+			Containers: []corev1.Container{
+				{
+					Name:    "infra",
+					Image:   "ubuntu:23.10",
```

Comment:
Why use a `ubuntu` image here, won't an `alpine` image be smaller and be able to perform the same task?

Reply:
Fair enough
Keeping consistency in mind then, maybe we should (in this PR or another) have all our helper pods use `ubuntu`. In this file itself there are two more jobs being deployed (one for zero downtime validation and another for a debug pod) using different images, so maybe this is a case to have those use `ubuntu` as well

Reply:
https://github.com/gardener/etcd-druid/issues/670